example_dags/example_load_file.py

```diff
@@ -128,4 +128,27 @@
         ),
     )
     # [END load_file_example_10]
+
+    # [START load_file_example_11]
+    aql.load_file(
+        input_file=File(
+            "s3://astro-sdk/sample_pattern", conn_id="aws_conn", filetype=FileType.CSV
+        ),
+        output_table=Table(conn_id="bigquery", metadata=Metadata(schema="astro")),
+        use_native_support=False,
+    )
+    # [END load_file_example_11]
+
+    # [START load_file_example_12]
+    aql.load_file(
+        input_file=File(
+            "gs://astro-sdk/workspace/sample_pattern",
```

Comment:
Can we have an example of using `*` wildcard here please @utkarsharma2

Reply:
@kaxil Sure, I'll add it.

Reply:
@kaxil the implementation we have only supported prefixes in the file path, I have updated the docs to reflect that.

https://cloud.google.com/storage/docs/listing-objects

Reply:
Created a separate PR to expose this feature - https://github.com/astronomer/astro-sdk/pull/691